security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisProvider.java

```diff
@@ -75,29 +70,14 @@ static List<SecurityAnalysisProvider> findAll() {
      *
      * @param network IIDM network on which the security analysis will be performed
      * @param workingVariantId network variant ID on which the analysis will be performed
-     * @param detector
-     * @param filter
-     * @param computationManager
-     * @param parameters specific security analysis parameters
      * @param contingenciesProvider provides list of contingencies
-     * @param interceptors
-     * @param monitors stateMonitor that defines the branch bus and threeWindingsTransformer about which informations will be written after security analysis
-     * @param limitReductions list of the limit reductions to apply
```

Comment:
transfer javadoc to SecurityAnalysisRunParameters (and maybe elaborate more than actual one?)